challenges/objects-arrays.js

```diff
@@ -97,23 +140,33 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
+const lowerCase = zooAnimals.map(function(lower){
+  return lower.animal_name.toLowerCase()
+});
+
+
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
+const largerPopulation = zooAnimals.filter(function(count) {
+  if (count.population < 5 ) {
+    return count
+  }
+});
```

Comment:
you do not have to use if statement with filter method - 
f/e: `largerPopulation = zooAnimals.filter(animal => animal.population < 5);`  ;)